### DISCUSSION TITLE
Spanish Localization project for quarkus.io website

### DISCUSSION THREAD

**Discussion by rdnovell** [quarkusio/quarkus#25124] [Community]
Hello guys,

I think could be nice to have a localized site: https://es.quarkus.io/. So I created a Spanish Localization demo project https://github.com/rdnovell/es.quarkus.io.

Is anyone else interested to contribute to a Spanish Localization website https://es.quarkus.io?

Thanks.

**Comment by maxandersen** [comment]
That would be cool! My spanish is limited to Hola! so I can't help on translation but if others are up for it I can definitely help enable and setup es.quarkus.io dns!

**Comment by ynojima** [comment]
Thank you for your interest to localizing quarkus.io.
I'm a maintainer of ja.quarkus.io and ask me anything about the utility scripts in the repo.

**Reply by rdnovell** [reply]
Hi Yoshikazu, about the deepL api key, how should be the setup? we can share the same from ja repo or we need a new one?

**Reply by ynojima** [reply]
I sent it on corp. Google Chat.

**Comment by aureamunoz** [comment]
I will be happy to help!!

**Comment by someth2say** [comment]
Count on me!

**Reply by juangon** [reply]
I will help too! Any starting point for translating?

**Comment by rdnovell** [comment]
Hello, thank you all for your comments. The site now is inside QuarkusIO organization [es.quarkus.io](https://github.com/quarkusio/es.quarkus.io), thanks @ynojima, @maxandersen for your help.
For contributors there's is a [guide](https://github.com/quarkusio/es.quarkus.io/blob/master/translation-guide.es.md) to help, but if you have any questions please ping me.
